EvidenceApi/DocumentTypes.json

```diff
@@ -128,6 +128,31 @@
         "id": "proof-of-pension",
```

Comment:
Confirmed that we will keep proof-of-pension as there are already entries in the db that use this id.